programs/programs/rental_assistance_grant/calculator.py

```diff
@@ -0,0 +1,48 @@
+from programs.programs.calc import ProgramCalculator, Eligibility
+import programs.programs.messages as messages
+from programs.co_county_zips import counties_from_zip
+from programs.sheets import sheets_get_data
+from integrations.util import Cache
+
+class RentalAssistanceGrant(ProgramCalculator):
+    amount = 10_000
+
```

Comment:
We have to add the dependencies, because we have a partner that uses our API to create a screen that they then share with their users. They don't have all of the data that we have, so we don't calculate benefits that require fields that are not included in the screen.

```suggestion
    dependencies = ['income_amount', 'income_frequency', 'household_size', 'zipcode']

```